docs/changehistory/NextVersion.md

```diff
@@ -1,5 +1,5 @@
 ---
-publish: false
+publish: true
```

Comment:
@evanpreslar-bentley the way it was previously is correct.  The version bump job actually fixed the way this was handled.

The change you'll want to make to fix the doc links is in `docs/changehistory/index.md` and `docs/changehistory/2.10.0.md`.

Reply:
Ah, I misunderstood. The publish fix is for master.
I'll get the failing docs updated, then